submissions/ustymchyk/html-css-popup/index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML and CSS popup</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav">
+      <a class="header__item" href>Mail</a>
+      <a class="header__item" href>Images</a>
+      <input type="checkbox" id="menu" class="header__item menu-btn">
```

Comment:
Make your menu with the same structure, for example:
```suggestion
     <ul>
        <li class="nav__item">
          <a class="header__item" href>Mail</a>
        </li>
        <li class="nav__item">
          <a class="header__item" href>Images</a>
        </li>
        <li class="nav__item">
          <input type="checkbox" id="menu" class="header__item menu-btn">
        </li>
      </ul>
```

Reply:
I thought it is not allowed to use a **div** inside a **li**. Thanks.

Reply:
It is valid to use a `div` inside a `li`, but in my example there is no such situation.

Reply:
In this situation i have to add .menu after input. In other case i can`t user :checked + .menu selector to open/hide menu. So, div with menu will be inside li.

Reply:
![image](https://user-images.githubusercontent.com/18123557/99302698-657f3280-2858-11eb-985d-9acaab3292f9.png)